Tasks/PublishSymbols/Helpers.ps1

```diff
@@ -97,25 +59,26 @@ function Add-DbghelpLibrary {
 function Add-SourceServerStream {
     [cmdletbinding()]
     param(
-        [string]$PdbStrPath = $(throw 'Missing PdbStrPath.'),
-        [string]$SymbolsFilePath = $(throw 'Missing SymbolsFilePath.'),
-        [string]$StreamContent = $(throw 'Missing StreamContent.')
+        [ValidateNotNullOrEmpty()]
+        [string]$PdbStrPath,
+
+        [ValidateNotNullOrEmpty()]
+        [string]$SymbolsFilePath,
+
+        [ValidateNotNullOrEmpty()]
+        [string]$StreamContent
     )
 
     # Create a temp file to store the stream content.
     Write-Verbose "Writing source server stream content to a temp file."
-    [string]$streamContentFilePath = [System.IO.Path]::GetTempFileName()
+    $streamContentFile = New-TemporaryFile
```

Comment:
I noticed New-TemporaryFile is new in PS5.0 will this matter? since we don't demand PS version on agent.